packages/web-react/src/components/Navigation/README.md

```diff
@@ -0,0 +1,122 @@
+# Navigation
+
+The `Navigation` component is a container for the navigation links of the application.
+
+It consists of a these parts:
+
+- [Navigation](#navigation)
+- [NavigationItem](#navigationitem)
+- [NavigationLink](#navigationlink)
+
+## Navigation
+
+The `Navigation` is a `nav` wrapper for navigation items.
+
+```jsx
+import { Navigation } from '@lmc-eu/spirit-web-react';
+
+<Navigation>{/* Navigation items go here */}</Navigation>;
+```
+
+It centers its children vertically and if the children are not `NavigationItem` components,
+it will apply a gap between them.
+
+### API
+
+| Name       | Type                                                                                | Default | Required | Description               |
+| ---------- | ----------------------------------------------------------------------------------- | ------- | -------- | ------------------------- |
+| `children` | `ReactElement<HTMLLIElement>`\|`ReactElement<NavigationItem>`\|Array of these types | `null`  | ✓        | Content of the Navigation |
+
+The components accept [additional attributes][readme-additional-attributes].
+If you need more control over the styling of a component, you can use [style props][readme-style-props]
+and [escape hatches][readme-escape-hatches].
+
+## Navigation Item
+
+The `NavigationItem` is a container for navigation links.
+
+```jsx
+import { NavigationItem } from '@lmc-eu/spirit-web-react';
+
+<NavigationItem>{/* Navigation links go here */}</NavigationItem>;
+```
+
+### API
+
+| Name       | Type                    | Default | Required | Description                   |
+| ---------- | ----------------------- | ------- | -------- | ----------------------------- |
+| `children` | `string` \| `ReactNode` | `null`  | ✓        | Content of the NavigationItem |
```

Comment:
We will need `elementType` for this component so we are able to serve cases with buttons instead of (navigable) links.

Reply:
I don't think so. By default, Navigation contain `ul` tag. So NavigationItem is basically `li` wrapper that accepts NavigationLink or buttons. That should be enough for the case you describe... or not?

Reply:
Yes, I think @adamkudrna misread and thought this is NavigationLink :)